### ISSUE TITLE
fast login on admin char → crush server

### ISSUE THREAD

**Issue by igroglaz** [draconisPW/PWMAngband#104]
when I logged in with admin fast (pushed enter several times fast, as I do often :E ) -  I've managed to crush the server

```
220320 124645 Welcome Admin=n@blackhole (*) (version 1400)
220320 124645 Shutting down (panic save).
220320 124645 Server panic info save succeeded!
```

Maybe it happens not only with admin. I'll test and report more.

**Comment by draconisPW** [comment]
Unfortunately I cannot reproduce this to fix the problem. This is probably due to lag and you would need to put the server on debug mode to get the line which produces the crash.

**Comment by igroglaz** [comment]
I had it only once for now.. I'll get close look on it.

**Comment by draconisPW** [comment]
Could you test this again with debug mode on the server?

**Comment by draconisPW** [comment]
Note: when you log in very fast with any character, there is a good chance to end up with a black screen which you need to refresh to be able to play. I'll try to also fix this when I have time.

**Comment by igroglaz** [comment]
yep, it's pretty often problem. I suppose we should just hardcode several delays between loading the char

**Comment by draconisPW** [comment]
This has been fixed with #392.
